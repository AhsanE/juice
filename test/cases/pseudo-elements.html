<html><body>
<a href="#">Test</a>
<b></b>
<b><span></span></b>
</body></html>
