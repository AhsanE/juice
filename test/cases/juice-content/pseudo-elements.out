<html><body>

<a href="#" style="color: blue; text-decoration: underline;"><span>Â®+</span>Test<span style="font-weight: bold;">a</span></a>
<b><span style="color: green;">b</span></b>
<b><span style="color: red;"></span><span style="color: green;">b</span></b>
</body></html>
